src/System.Linq.Expressions/tests/ExceptionHandling/ExceptionHandlingExpressions.cs

```diff
@@ -288,93 +282,60 @@ public void NonExceptionDerivedExceptionWrappedInterpreted()
             Assert.Equal("System.Runtime.CompilerServices.RuntimeWrappedException", ex.GetType().FullName);
```

Comment:
This reminds me. What reference would be needed to use the type `RuntimeWrappedException`? That's how this should be done, really.